### ISSUE TITLE
Option to specify file type

### ISSUE THREAD

**Issue by nu8** [mholt/archiver#245]
This file:

https://github.com/microsoft/terminal/releases/download/v1.3.2651.0/Microsoft.WindowsTerminal_1.3.2651.0_8wekyb3d8bbwe.msixbundle

is essentially just a Zip file, with a different extension. However it seems it
is not possible to extract it as is:

~~~
PS C:\> arc unarchive Microsoft.WindowsTerminal_1.3.2651.0_8wekyb3d8bbwe.msixbundle
format unrecognized by filename: Microsoft.WindowsTerminal_1.3.2651.0_8wekyb3d8bbwe.msixbundle
~~~

but if I rename it to `.zip`, it works fine. It seems Arc should either have a
heuristic to detect file type, or allow user to specify file type.

**Comment by coolaj86** [comment]
I like this.

**Comment by BatoolMM** [comment]
I'd like to give it a try, can you assign it to me, please?

**Comment by coolaj86** [comment]
@BatoolMM Assigned!

**Comment by BatoolMM** [comment]
@coolaj86 , sorry, I won't be able to get it done this week, so I removed my assignment in case others (in the hacktoberfest) can get it done way quicker. Sorry again.

**Comment by Red-0111** [comment]
@coolaj86 please assign will try my best

**Comment by coolaj86** [comment]
@Red-0111 Re-assigned. Take it.

**Comment by Red-0111** [comment]
Ok

**Comment by nu8** [comment]
Using the patch seems to work as expected:

~~~
arc -ext zip unarchive Microsoft.WindowsTerminal_1.3.2651.0_8wekyb3d8bbwe.msixbundle
~~~

**Comment by coolaj86** [comment]
Fixed in https://github.com/mholt/archiver/pull/252. Released with v3.4.0
